### ISSUE TITLE
Redirecting user to original page after auth

### ISSUE THREAD

**Issue by hownowbrowncow** [hslavich/OneloginSamlBundle#79]
Hi,

I'm trying to figure out how i can redirect the user to the page they originally came from instead of the root page. It seems the `relayState` is always set to `/saml/login`. This could be set in the controller while doing `$auth->login($relayStateUrl)` but even if i hard code something in there it's not redirecting me to the correct page. Is this a problem with my config or is it a feature missing from this package? I'm a bit new to symfony and how firewalls work so it's possible i'm overlooking something very simple on my end.

Any help would be appreciated.

```
security:
  providers:
    saml_provider:
      saml:
        user_class: 'App\Security\User'
        default_roles: ['ROLE_USER']

  firewalls:
    dev:
      pattern: ^/(_(profiler|wdt)|css|images|js)/
      security: false
    main:
      anonymous: true
      pattern: ^/
      saml:
        username_attribute: uid
        check_path: /saml/acs
        login_path: /saml/login
      logout:
        path: /saml/logout

  access_control:
    - { path: ^/saml/login, roles: IS_AUTHENTICATED_ANONYMOUSLY }
    - { path: ^/saml/metadata, roles: IS_AUTHENTICATED_ANONYMOUSLY }
    - { path: ^/, roles: ROLE_USER }
```

**Comment by hslavich** [comment]
It seems a missing feature, I will try to fix this.
Thanks

**Comment by hownowbrowncow** [comment]
Also to add to this, i'm currently having an issue where I need to append addition query params during the login process for my IDP to work correctly. The func signature for login seems to allow this as seen [here](https://github.com/onelogin/php-saml/blob/master/lib/Saml2/Auth.php#L473).

I'll gladly try resolve some of these issues this weekend if you don't have time. I'm a bit new to symfony so i might not come up with the best solutions but i'm always willing to try help.

**Comment by a-menshchikov** [comment]
Solved by #105 
If you need to ignore RelayState and always redirect to target_path, just use default success handler instead of the one from bundle.
